arm/config/config.py

```diff
@@ -15,7 +15,15 @@ def _load_config(fp):
     return config
 
 
+def load_abcde(fp):
```

Comment:
Why is there an argument/variable being passed in that is never used ?

Reply:
didn't know you could reference code within comments, that's pretty need.

good pickup, error on my part. fixed with the latest commit to use 'fp'